indico/web/client/js/react/components/Datepicker.jsx

```diff
@@ -0,0 +1,83 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {Param, Translate} from 'indico/react/i18n';
+
+import 'indico/custom_elements/ind_datepicker';
+
+export default function Datepicker({format, onChange, value, ...inputProps}) {
```

Comment:
I'd prefer calling it `DatePicker` instead of `Datepicker` for the sake of consistency with existing capitalization of `DatePicker` in the codebase.